build.sbt

```diff
@@ -13,45 +13,37 @@ val appName = "help-to-save-stride-frontend"
 lazy val plugins: Seq[Plugins] = Seq.empty
 lazy val playSettings: Seq[Setting[_]] = Seq.empty
 
-val akkaVersion     = "2.5.23"
-
-val akkaHttpVersion = "10.0.15"
-
-
-dependencyOverrides += "com.typesafe.akka" %% "akka-stream"    % akkaVersion
-
-dependencyOverrides += "com.typesafe.akka" %% "akka-protobuf"  % akkaVersion
-
-dependencyOverrides += "com.typesafe.akka" %% "akka-slf4j"     % akkaVersion
-
-dependencyOverrides += "com.typesafe.akka" %% "akka-actor"     % akkaVersion
-
-dependencyOverrides += "com.typesafe.akka" %% "akka-http-core" % akkaHttpVersion
-
 lazy val dependencies = Seq(
   ws,
-  "uk.gov.hmrc" %% "govuk-template" % "5.66.0-play-26",
-  "uk.gov.hmrc" %% "mongo-caching" % "7.0.0-play-26",
-  "uk.gov.hmrc" %% "play-ui" % "9.2.0-play-26",
-  "uk.gov.hmrc" %% "bootstrap-frontend-play-26" % "5.2.0",
-  "uk.gov.hmrc" %% "domain" % "5.11.0-play-26",
+  "uk.gov.hmrc" %% "govuk-template" % "5.72.0-play-28",
+  "uk.gov.hmrc" %% "mongo-caching" % "7.0.0-play-28",
+  "uk.gov.hmrc" %% "play-ui" % "9.7.0-play-28",
+  "uk.gov.hmrc" %% "bootstrap-frontend-play-28" % "5.2.0",
+  "uk.gov.hmrc" %% "domain" % "6.2.0-play-28",
   "org.typelevel" %% "cats-core" % "2.6.0",
   "com.github.kxbmap" %% "configs" % "0.6.1",
   compilerPlugin("com.github.ghik" % "silencer-plugin" % "1.7.1" cross CrossVersion.full),
   "com.github.ghik" % "silencer-lib" % "1.7.1" % Provided cross CrossVersion.full
```

Comment:
It's a minor change, not a mandatory.
If possible change silencer version to 1.7.5 for both entries.

Reply:
done